### PR TITLE
Use mediumblob for session data storage (#3006)

### PR DESCRIPTION
**Pull request by jekkos**
Use MEDIUMBLOB type for session data storage. This should increase the max amount of items that can be stored in a cart.

### CODE REVIEW

File: application/migrations/20200819000000_modify_session_datatype.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php if (!defined('BASEPATH')) exit('No direct script access allowed');
+
+class Migration_modify_session_datatype extends CI_Migration
```
Comment (daN4cat) [inline application/migrations/20200819000000_modify_session_datatype.php:3]: date of the file name is wrong, shouldn't it be 20210102000000....

**Review by daN4cat — CHANGES_REQUESTED** [review]
See my review comment

File: application/migrations/20200819000000_modify_session_datatype.php (hunk 1 above)
Comment (jekkos) [reply application/migrations/20200819000000_modify_session_datatype.php]: Ok filename adapted.

**Comment by jekkos** [comment]
@daN4cat should be good now

**Pull request merged**